docker/development/Dockerfile

```
@@ -30,7 +30,7 @@ RUN apt-get update -y \
 
 ## Open Space Toolkit ▸ Core
 
-ARG OSTK_CORE_VERSION="0.7.3"
+ARG OSTK_CORE_VERSION="1.0.0"
```

can't you just do this here?
```suggestion
ARG OSTK_CORE_VERSION="1.0"
```

---

Nope. This doesn't have @kyle-cochran  fancy auto fetching script. Will add that in a separate MR.

---

ah right I forgot this is cpp and not python.